app/design/frontend/base/default/template/algoliasearch/internals/configuration.phtml

```diff
@@ -119,15 +130,17 @@ if ($config->isInstantEnabled()) {
         $isSearchPage = true;
     }
 
+    foreach ($facets as $facet) {
```

Comment:
Not sure about it as it's harder to read and I had to think for a while what your code does.
And it would be less efficient as it'd always iterate through all `$facets` and then again until it finds the correct attribute.